src/components/popUp/index.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState, useEffect } from 'react';
+import Styles from './styles.module.css';
+import IconButton from '../iconButton';
+import Button from '../button'
+import SignIn from '../signInForm'
+import SignUp from '../SignUp'
+
+const Popup = ({ isOpen, onClose, isNewUser, setNewUser}) => {
+
+    const changeToSignIn = ()=>{
```

Comment:
Why not instead of these two functions :
onClick={()=> setNewUser( true )}
onClick={()=> setNewUser( false )}